src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -224,7 +227,7 @@ public class SegueGuiceConfigurationModule extends AbstractModule implements Ser
   private static IGroupObserver groupObserver = null;
 
   private static Collection<Class<? extends ServletContextListener>> contextListeners;
-  private static final Map<String, Reflections> REFLECTIONS = com.google.common.collect.Maps.newHashMap();
+  private static final Map<String, List<Class<?>>> REFLECTIONS = new HashMap<>();
```

Comment:
This is no longer the right name, since the map values are not `Reflections`, but a list of classes... Maybe `classesByPackage` could be a more descriptive name (providing that's what this variable is about).